app/models/author.rb

```diff
@@ -1,2 +1,20 @@
 class Author < ApplicationRecord
+    has_many :tweets
+    has_many :bookmarks
+    has_many :likes
+    has_and_belongs_to_many :followers
+
```

Comment:
Where are the followees? tweeter also followees not just followers

![image](https://github.com/franciscoWork/xtwitter/assets/1312458/9ca43c91-ce4f-4eb9-ac56-5dbd19dcc2c1)